app/models/miq_action.rb

```diff
@@ -616,8 +616,7 @@ def action_delete_snapshots_by_age(action, rec, _inputs)
     age_threshold = (Time.now.utc - action.options[:age])
     has_ch = false
     snaps_to_delete = rec.snapshots.each_with_object([]) do |s, arr|
-      has_ch = true if s.is_a_type?(:consolidate_helper)
-      next if s.is_a_type?(:evm_snapshot) || s.is_a_type?(:vcb_snapshot)
```

Comment:
Is the vcb_snapshot conditional check pushed down somewhere else or just not needed?

Reply:
https://github.com/ManageIQ/manageiq/pull/19551#issuecomment-557637176

I think I'll just move that logic into the vmware provider

Reply:
https://github.com/ManageIQ/manageiq-providers-vmware/pull/485